owncloudApp/src/main/java/com/owncloud/android/presentation/ui/security/PassCodeActivity.kt

```diff
@@ -282,10 +290,38 @@ class PassCodeActivity : AppCompatActivity() {
         }
     }
 
+    private fun lockScreen() {
+        val timeToUnlock = passCodeViewModel.getTimeToUnlockLeft()
+        if (timeToUnlock > 0) {
+            binding.lockTime.isVisible = true
+            for (editText: EditText? in passCodeEditTexts) {
+                editText?.isEnabled = false
+            }
+            object : CountDownTimer(timeToUnlock, 1000) {
+                override fun onTick(millisUntilFinished: Long) {
+                    val minutes = ((millisUntilFinished + 1000) / 1000) / 60
+                    val seconds = ((millisUntilFinished + 1000) / 1000) % 60
+                    val timeString = String.format("%02d:%02d", minutes, seconds)
```

Comment:
I would try to define these values in an extension function, maybe in a `FormatDatesExt.kt` file, what do you think?